server/lib/awsS3.js

```diff
@@ -0,0 +1,14 @@
+import S3 from 'aws-sdk/clients/s3';
+import config from 'config';
+
+// Create S3 service object & set credentials and region
+let s3;
+if (config.aws.s3.key) {
+  s3 = new S3({
+    accessKeyId: config.aws.s3.key,
+    secretAccessKey: config.aws.s3.secret,
+    apiVersion: '2006-03-01',
+    region: 'us-west-1',
```

Comment:
Let's make it nice and have `apiVersion` and `region` configurable.

Add an environment variable here:
- https://github.com/opencollective/opencollective-api/blob/master/config/custom-environment-variables.json

And current values as default here:
- https://github.com/opencollective/opencollective-api/blob/master/config/default.json

Reply:
good point, I'll get on this